vj4/locale/zh_CN.yaml

```diff
@@ -439,7 +438,7 @@ Delete own discussion replies: 删除讨论中自己的回复
 Delete discussion replies of own discussion: 删除自己讨论中的回复
 View contests: 查看比赛
 View contest scoreboard: 查看比赛成绩表
-View hidden contest scoreboard: 查看隐藏的比赛成绩表
+View hidden contest submission status and scoreboard: 查看隐藏的比赛递交状态和成绩表
```

Comment:
A hidden scoreboard implies hidden 'submission status'es.

Reply:
nop. only the opposite is true: when a scoreboard is visible, its submission status is visible.